bin.src/generateCoefficients.py

```diff
@@ -87,25 +87,33 @@
     else:
         tStart = args.tStart
 
+    # We have already ruled out the case where args.tSpan and args.tEnd are None, above.
```

Comment:
Move all of this code up to just below where we test that tStart and tEnd are not both None (just so that all of the args.t* tests are in the same place)